Tasks/google-play-release/GooglePlay.js

```diff
@@ -27,6 +27,17 @@ if (authType === "JsonFile") {
 }
 
 var apkFile = resolveGlobPath(tl.getPathInput("apkFile", true));
+var apkFileList = [apkFile];
+var additionaApks = tl.getDelimitedInput('additionalApks', '\n');
+if (additionaApks.length > 0) {
```

Comment:
Is `additionalApks` an array? If so, we could just use contact the `apkFileList` with it instead of needing to enumerate/push it.